src/config/Config.cpp

```diff
@@ -226,6 +226,16 @@ AudioBackend Config::GetCurrentAudioBackend() {
     return AudioBackend::SDL;
 }
 
+AudioChannelsSetting Config::GetCurrentAudioSurround() {
+    int surround = GetInt("CVars.gAudioChannelsSetting");
+    switch (surround) {
+        case 1:
+            return AudioChannelsSetting::surround51;
+        default:
+            return AudioChannelsSetting::stereo;
+    }
```

Comment:
we should avoid magic numbers here

```suggestion
    switch (surround) {
        case AudioChannelsSetting::surround51:
            return AudioChannelsSetting::surround51;
        case AudioChannelsSetting::stereo:
        case AudioChannelsSetting::max:
        default:
            return AudioChannelsSetting::stereo;
    }
```

it's also possible we could just simplify this entire method to just `static_cast` directly to the enum from the int

```cpp
AudioChannelsSetting Config::GetCurrentAudioSurround() {
    return static_cast<AudioChannelsSetting>(GetInt(CVAR_AUDIO_CHANNELS_SETTING, static_cast<int32_t>(AudioChannelsSetting::max));
}
```

it's also possible implicit casting will work too but i'm not sure what rules all the ports set up for all the different compilers and if any might complain about it

```cpp
AudioChannelsSetting Config::GetCurrentAudioSurround() {
    return GetInt(CVAR_AUDIO_CHANNELS_SETTING, static_cast<int32_t>(AudioChannelsSetting::max));
}
```